dev/ComboBox/ComboBoxAttachedProperties.cpp

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <common.h>
+#include "ComboBoxAttachedProperties.h"
+#include "ComboBoxAttachedProperties.properties.h"
+
+static constexpr auto c_popupName = L"Popup"sv;
+static constexpr auto c_popupBorderName = L"PopupBorder"sv;
+static constexpr auto c_editableTextName = L"EditableText"sv;
+
+// Normal ComboBox and editable ComboBox have differnt CornerRadius behaviors.
+// Xaml is not lifted yet when we implementing this feature so we don't have access to ComboBox code.
+// Creating this attached property to help us plug in some extra logic without touching the actual ComboBox code.
+void ComboBoxAttachedProperties::OnApplyDynamicCornerRadiusPropertyChanged(
+    const winrt::DependencyObject& sender,
+    const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    bool shouldMonitorDropDownState = unbox_value<bool>(args.NewValue());
+    if (shouldMonitorDropDownState)
+    {
+        if (auto comboBox = sender.try_as<winrt::ComboBox>())
+        {
+            comboBox.DropDownOpened(&ComboBoxAttachedProperties::OnDropDownOpened);
```

Comment:
These will need to be removed at some point. At the least, these will need to be auto_revoke.

Also when the value is toggled to false, they should be detached.